api/interestr/website/views.py

```diff
@@ -180,6 +180,12 @@ def get(self, request):
             users = users.filter(Q(username__icontains=query)).distinct()
         return render(request, self.template_name, {'groups': groups, 'users': users, 'search_term': query})
 
+class SearchAdvancedView(LoginRequiredMixin, generic.ListView):
+    template_name = 'website/search_advanced.html'
+
+    def get(self, request):
+        data_templates = [data_template for group in request.user.joined_groups.all() for data_template in group.data_templates.all()]
```

Comment:
Sounds good, changes.

Reply:
Also, to avoid name clashes from groups (groups can have templates with the same name), I included the group name in the select options:
<img width="353" alt="screen shot 2017-12-27 at 01 01 34" src="https://user-images.githubusercontent.com/13895224/34365469-8ed5c29a-eaa1-11e7-9aa1-0789d47f3d0a.png">